script/metalsmith/appIncluder.js

```diff
@@ -0,0 +1,29 @@
+const request = require('request');
```

Comment:
needs ❤️. But not _too_ much for the first iteration. Let's just get the spike out and address issues as they pop up.